src/jepsen/dqlite.clj

```diff
@@ -25,16 +27,27 @@
    :bank   bank/workload
    :set    set/workload})
 
-(defn assertion-checker
+(def assertion-pattern
+  "An egrep pattern for finding assertion errors in log files."
+  "Assertion|raft_start|start-stop-daemon|for jepsen")
+
+(defn core-dumps-checker
   []
   (reify checker/Checker
     (check [this test history opts]
-      (if-let [crashes (merge
-                        (db/logged-assertions test)
-                        (db/core-dumps test))]
-        {:valid? false
-         :crashes crashes}
-        {:valid? true}))))
+      (let [blips
+            (->> (:nodes test)
+                 (pmap (fn [node]
+                         (let [{:keys [exit]}
+                               (->> (store/path test node "data/core")
```

Comment:
Hmm, I need to check that Jepsen will actually copy the core dump files over if they exist remotely

Reply:
It seems not, or the core dump is not being created in the expected place. Trying to fix that now